proto/nameserver2.proto

```diff
@@ -472,6 +481,8 @@ message ReFreshSessionRequest {
     optional string     clientVersion = 6;
     optional string     clientIP = 7;
     optional uint32     clientPort = 8;
+    optional string     clientuuid = 9;
+    optional bool       iswriter = 10;
```

Comment:
ditto, and it's better to encapsulation `uuid` and `openflags` into a separate class, like

```
message OpenFileConext {
    required uint32 openflags = 1;
    required string owner/openid/uuid = 2;
};
```

Reply:
fix